packages/react/src/generators/application/lib/add-e2e.ts

```diff
@@ -0,0 +1,61 @@
+import type { GeneratorCallback, Tree } from '@nx/devkit';
+import {
+  addProjectConfiguration,
+  ensurePackage,
+  getPackageManagerCommand,
+  joinPathFragments,
+  readProjectConfiguration,
+} from '@nx/devkit';
+import { webStaticServeGenerator } from '@nx/web';
+
+import { nxVersion } from '../../../utils/versions';
+import { NormalizedSchema } from '../schema';
+
+export async function addE2e(
+  tree: Tree,
+  options: NormalizedSchema
+): Promise<GeneratorCallback> {
+  if (options.e2eTestRunner === 'none') {
```

Comment:
small nitpick here I would make this a switch statement instead of an `if else` but that's a personal choice

Reply:
changed